README.md

```diff
@@ -92,6 +92,25 @@ The demo server applications have been deployed and are available at the followi
 
 The following section will get you up to speed on the eWallet and show you how to deploy it in local.
 
+## Using Docker image
+
+We provides a Docker image for quick installation at the [omisego/ewallet](https://hub.docker.com/r/omisego/ewallet/) repository on Docker Hub with the following tags:
+
+* `latest` tracks current (i.e. develop branch)
+* `stable` tracks stable (i.e. master branch)
+
+Additionally, all commits in develop and master branches are also built. It is highly recommend to pin the version to certain commit in anything that resembles production environment. An easiest way to get the Docker image running would be (assuming [PostgreSQL]() is already setup):
```

Comment:
`highly recommended` - `to certain commits` (`specific` might be more appropriate than `certain`)

Reply:
Link for PostgreSQL missing.

Reply:
👌